### DISCUSSION TITLE
css-map

### DISCUSSION THREAD

**Discussion by ohitstom** [Delusoire/bespoke#13] [General]
will this still be implemented, looking to start porting Comfy.

**Comment by Delusoire** [comment]
The css-map isn't applied to the client, instead it's applied to the modules themselves using `bun run build:modules`

**Reply by ohitstom** [reply]
any chance this can be added to the install guide I had no idea sorry

also that kinda sounds like hell to maintain for theme Devs unless I'm misunderstanding? like how am I supposed to theme a new thing without manually checking the CSS map that does not sound fun in the slightest 

maybe when running bespoke Dev we could patch the client (I'm begging)
